qa/integration-tests/src/test/java/io/zeebe/broker/it/DataDeleteTest.java

```diff
@@ -44,6 +44,7 @@ public static void configureCustomExporter(final BrokerCfg brokerCfg) {
     data.setMaxSnapshots(MAX_SNAPSHOTS);
     data.setSnapshotPeriod(SNAPSHOT_PERIOD_SECONDS + "s");
     data.setLogSegmentSize("8k");
+    brokerCfg.getNetwork().setMaxMessageSize("8K");
```

Comment:
I had expect it more on data section as record size but yeah maybe it is more intuitive as message size :man_shrugging:

Reply:
Yeah, it is used for many things :sweat_smile: Max fragment size (i.e. record/batch size), transport max message size, Atomix entry size etc.